parm/config/gfs/config.resources

```diff
@@ -692,9 +706,13 @@ case ${step} in
       "C48" | "C96")
         export npe_upp=${CASE:1}
       ;;
-      "C192" | "C384" | "C768")
+      "C192" | "C384")
+        export npe_upp=120
+        export memory_upp="96GB"
+      ;;
+      "C768")
         export npe_upp=120
-        export memory_upp="48GB"
+        if [[ ${machine} == "WCOSS2" ]]; then export memory_upp="480GB" ; fi
```

Comment:
What about the memory value for the other machines for C768? Run upp exclusively elsewhere?

I don't see memory for the lowest resolutions, do we need that too or also run exclusively?

Reply:
how about now.  This should atleast cover what was there before.

Reply:
Works for me, thanks @aerorahul !